common/subset-tests.js

```diff
@@ -0,0 +1,27 @@
+// Only test a subset of tests with, e.g., ?1-10 in the URL
+// Sample usage:
+// let currentSubTest = 0;
+// for (const test of tests) {
+//   currentSubTest++;
+//   if (shouldRunSubTest(currentSubTest)) {
+//     ... run the test ...
+//   }
+// }
+(function() {
+  var subTestStart = 0;
+  var subTestEnd = Infinity;
+  var match;
+  if (location.search) {
+      match = /^\?(\d+)-(\d+|last)(?:&|$)/.exec(location.search);
+      if (match) {
+        subTestStart = match[1];
```

Comment:
nit: parseFloat(match[1]) or parseInt(match[1], 10)

Reply:
rather than relying on string/number comparison later on. (Of course, it's all per spec, so totally optional)